edge_orchestrator/edge_orchestrator/domain/use_cases/supervisor.py

```diff
@@ -55,8 +55,9 @@ async def wrapper(item: Item, *args):
 
     async def inspect(self, item: Item):
 
-        item.station_config = self.station_config.active_config_name
-        self.edge_station.register_cameras(self.station_config)
+        if self.station_config.active_config is not None:
+            item.station_config = self.station_config.active_config_name
+            self.edge_station.register_cameras(self.station_config)
 
```

Comment:
ce serait bien d'ajouter un else au moins avec un log

Reply:
finalement c'est nécessaire d'ajouter if self.station_config.active_config is not None ?

Reply:
J'avais ajouté ça dans mes débuts quand je manquais de compréhension du code : quand j'ai commencé à tester ma route upload dans le front je n'allais pas set une config et du coup je me tapais une erreur dans le back
Je rajoute un log "Warning: config not setup" ?

Reply:
Oui après avoir retester c'est bien ça: j'aimerais ne pas pas avoir besoin de setup une config pour envoyer une photo dans le cloud. A voir comment on peut faire mieux les choses

Reply:
Ce que je propose c'est soit la feature est nécessaire dans ce cas on explicite le cas if et else si la feature n'est pas utilisée on retire la condition

Reply:
Ok problème réglé, on crée un nouveau use case : uploader